pkgs/development/haskell-modules/configuration-ghc-8.4.x.nix

```diff
@@ -385,6 +385,23 @@ self: super: {
     };
   });
 
+  ## Unmerged.  PR: https://github.com/sol/hpack/pull/277
+  ## Issue: https://github.com/sol/hpack/issues/276
+  hpack = overrideCabal super.hpack (drv: {
```

Comment:
Using `hpack_0_27_0` would not fix that issue?

Reply:
@peti, the unmerged PR is on top of `0.27`, so the instances simply do not exist -- the base for the PR is `340215bc936108edaa37fafa17ba71fc0550478a`, authored `Simon Hengel <sol@typeful.net>  2018-02-25 08:45:46`, so the very latest tip of upstream's `master`.